BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.m

```diff
@@ -954,6 +992,13 @@ - (NSString *)filePathOfUserSessionTaskGivenUserId:(NSString *)userId
     return [path stringByAppendingPathComponent:fileName];
 }
 
+// Return dir path users/$userId/$associateId/started
```

Comment:
This feels like a comment which will be prone to getting out of date, and doesn't elucidate much.

Reply:
removed